stdlib/pagerduty/pagerduty.flux

```diff
@@ -10,6 +10,18 @@ builtin dedupKey
 option defaultURL = "https://events.pagerduty.com/v2/enqueue"
 
 
+// severity levels on status objects can be one of the following: ok,info,warn,crit,unknown
+// but pagerduty only accepts critical, error, warning or info.
+// formatSeverity turns a level from the status object into a pagerduty severity
+formatSeverity = (severity) => {
+    sev = strings.toLower(v:severity) 
+    if sev == "warn" then "warning"
+    else if sev == "crit" then "critical"
+    else if sev == "info" then "info"
+    else if sev == "ok" then "info"
+    else "unknown"
+}
+
 // `actionFromSeverity` converts a severity to an action; "ok" becomes "resolve" everything else converts to "trigger".
 actionFromSeverity = (severity)=> if strings.toLower(v:severity) == "ok" then "resolve" else "trigger"
```

Comment:
Should this function change now? It seems like this function should be called `actionFromLevel` to be consistent with the above changes. 

```
actionFromLevel = (level) => ...
```

Also note that its not possible to define `actionFromSeverity` since both `ok` and `info` levels are converted to the `info` severity.